docs/_includes/projects/docs/templates1/my-element.js

```diff
@@ -0,0 +1,31 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+class MyElement extends LitElement {
+  static get properties(){
+    return {
+      prop1: Number,
+      prop2: Number
+    };
+  }
+  constructor(){
+    super();
+    this.prop1=0;
+    this.prop2=1;
+  }
+  render(){
+    return html`
+      <p>Static paragraph.</p>
+      <p>prop1: ${this.prop1}</p>
+      <button @click="${() => this.changeMe('prop1')}">change prop1</button>
+      <button @click="${() => this.changeMe('prop2')}">change prop2</button>
+    `;
+  }
+  updated(){
+    console.log('Updated');
+  }
+  changeMe(prop){
+    var rand = Math.floor(Math.random()*100);
+    prop=='prop1'?this.prop1=rand:this.prop2=rand;
```

Comment:
Should either be an if statement or can be simplified into:
```suggestion
    this[prop] = rand;
```